### ISSUE TITLE
scrollbar-content has parent classes

### ISSUE THREAD

**Issue by gitbejbe1** [gromo/jquery.scrollbar#121]
Why your plugin copy classes from main container to children "scrollbar-content" ??? 

[https://postimg.org/image/q4h64toix/](url)

this is very irritating with planing css structure

**Comment by gromo** [comment]
By the main idea you shouldn't worry about CSS structure - classes are duplicated to content container to apply paddings, colors, background, etc... to content itself.

**Comment by gitbejbe1** [comment]
im working on latest bootstrap 3. I want make panel-body scrollable, so a write code:
```
<div class="panel">
      <div class="panel-body scrollbar-inner" style="max-height: 250px">
             <p>.....</p>
             <p>....</p>            
              [...]   
      </div>
</div>
```
I cant do this, because "panel-body" get styles only as "panel" children (just as in many other cases). "max-height" style not work. To define scroll content height i must set this styles inside scroll-content.

**Comment by gromo** [comment]
Then just update your CSS

**Comment by gitbejbe1** [comment]
i resolve it by adding new option (hight) in your plugin. After init, 'scrollbar-wrapper' get max-height style with specific value. 

Last question: 'scrollbar-wrapper' must have padding: 0 ? If i remove this from your styles, scrollbar will calculate correct without it?

**Comment by gromo** [comment]
It should be 0, and if you want padding, apply it to your container class - that's why classes applied for content itself. Also you can always set styles using additional classes for you content or add additional div wrapper into panel-body.

**Comment by strarsis** [comment]
Yes, I also have an issue with inner element having the same parent CSS class.
I have to use an inner wrapper now - duplicating styles by using same parent class can be convenient sometimes - but when the outside scrollbar element has been styled with shadow, etc - this will be shown twice, nested inside.

**Comment by gromo** [comment]
you can always set styles via CSS both for inner/outer CSS classes, inherited structures and all other cases - CSS is very powerful tool.

**Comment by strarsis** [comment]
@gromo: What selector is suitable for e.g. only selecting the inner element with same (parent) class?

**Comment by gromo** [comment]
.your-class { your styles }
.your-class.scroll-wrapper { disable your styles }

**Comment by strarsis** [comment]
@gromo: Thanks. This would work - though I like to avoid resetting styles.

**Comment by gromo** [comment]
you can update jquery.scrollbar.css file where all styles are reset for .scroll-wrapper via !important, so you can add there styles I missed
